net.ankiweb.Anki.json

```diff
@@ -0,0 +1,117 @@
+{
+  "app-id": "net.ankiweb.Anki",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
```

Comment:
Btw, 5.10 is out.

Reply:
Anki requires 5.9.x
https://github.com/dae/anki/blob/master/aqt/qt.py#L43